.github/workflows/build-and-push-assets.yml

```
@@ -137,7 +147,21 @@ jobs:
       TAG_BRANCH_NAME: ''                              # we'll override if the push is for tag
       LOCK_FILE: ''                                    # we'll override after checking files
       NO_CHANGES: ''                                   # we'll override if no changes to commit
+      BUILT_BRANCH_SUFFIX: '-built'
+      RELEASE_BRANCH_NAME: 'release'
+      USE_BUILT_BRANCH: ${{ inputs.USE_BUILT_BRANCH }}
+      USE_RELEASE_BRANCH: ${{ inputs.USE_RELEASE_BRANCH }}
     steps:
+      - name: BUILT_BRANCH_SUFFIX deprecation notice
+        if: ${{ inputs.BUILT_BRANCH_SUFFIX != '' }}
+        run: |
+          echo "::notice::The BUILT_BRANCH_SUFFIX variable is deprecated and will be removed soon. If you use it, please update your workflow to use USE_BUILT_BRANCH."
+          echo "USE_BUILT_BRANCH=true" >> $GITHUB_ENV
```

In https://github.com/inpsyde/reusable-workflows/pull/69/files#diff-931bfdfcfdadfc9685f2d544c3d0f533098b1b58e0f9fb070d747869971de4aaR48-R51, we did this slightly differently:
```suggestion
      - name: Deprecation warning
        if: ${{ inputs.BUILT_BRANCH_SUFFIX }}
        run: |
          echo "::warning::The BUILT_BRANCH_SUFFIX input is deprecated and will be removed soon. If you use it, please update your workflow to use USE_BUILT_BRANCH."
          echo "USE_BUILT_BRANCH=true" >> $GITHUB_ENV

```